source/routing/raptor_api.cpp

```diff
@@ -1191,20 +1191,23 @@ template<typename T>
 static void add_origin_destination(PbCreator& pb_creator,
```

Comment:
As you also set the requested_date_time, you should change the name of this function to something more meaningful.